docs/pages/reference/cli.mdx

```diff
@@ -363,6 +463,44 @@ $ tsh proxy --port 10700 app grafana &
 $ curl http://127.0.0.1:10700/api/users
 ```
 
+### tsh proxy aws
+
+Start local proxy for AWS access.  The user should already be logged into
+at least one AWS application.
+
+```code
+$ tsh proxy aws [<flags>] 
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
+| - | - | - | - |
+| `--app` | Currently logged in AWS app | AWS app| Optional Name of the AWS application to use if logged into multiple |
```

Comment:
I'm not sure what an example of an allowed value would be here. Are there any restrictions on formatting of the application name? What would be the source of truth for the application name?

Reply:
@ptgott can we mark this as resolved now?